### PR TITLE
solution to hw-5.3

### PR DESCRIPTION
**Pull request by ggazila**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -9,6 +16,37 @@
...
-        return null;
+        String stringsFromFile = "";
+        try {
+            stringsFromFile = Files.readString(Paths.get(fileName));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        String[] result = stringsFromFile.toLowerCase().replaceAll("[^A-Za-z]", " ").split(" ");
+        if (result.length < 1) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FileWork.java]: do we need this checking?

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -9,6 +16,37 @@
...
+        List<String> wordsWithW = new ArrayList<>();
+
+        for (String word : result) {
+            if (word.startsWith("w")) {
+                wordsWithW.add(word);
+            }
+        }
+
+        Collections.sort(wordsWithW);
+        result = convertToString(wordsWithW);
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FileWork.java]: try using standard method `toArray(...)` and removing convertToString()

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -9,6 +16,37 @@
...
+
+        String[] result = stringsFromFile.toLowerCase().replaceAll("[^A-Za-z]", " ").split(" ");
+        if (result.length < 1) {
+            return result;
+        }
+
+        List<String> wordsWithW = new ArrayList<>();
+
+        for (String word : result) {
+            if (word.startsWith("w")) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/FileWork.java]: make constant for `w`

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (ggazila) [reply src/main/java/core/basesyntax/FileWork.java]: No, my bad:)
I'll fix it

**Pull request closed without merging**